src/common/theme/components/HeadingTheme.ts

```diff
@@ -0,0 +1,15 @@
+import { defineStyleConfig } from '@chakra-ui/react';
+
+export const HeadingTheme = defineStyleConfig({
+  baseStyle: {
+    fontWeight: 'medium',
+  },
+  sizes: {
+    md: {
+      fontSize: { base: '2xl', md: '4xl' },
```

Comment:
i think the responsiveness should not be set in the theme section 🤔 . it would be confusing during development if we want to do `{{base: x , md: y}}` but turns out it's already set in the theme

Reply:
most of the font sizes i see in Figma are sized like that, you can still override it if there's a custom text. If we don't do the responsive style here, many Texts will have repeated fontSize prop. Previously there's a custom Text component but that may confused devs as well

Reply:
alrightt, thanks for the explanationn